src/Adapter/MySQL.php

```diff
@@ -324,7 +328,7 @@ protected function getFieldMapping()
                     (sp.from = \'0000-00-00 00:00:00\' OR \'' . date('Y-m-d H:i:s') . '\' >= sp.from) AND 
                     (sp.to = \'0000-00-00 00:00:00\' OR \'' . date('Y-m-d H:i:s') . '\' <= sp.to) 
                 )',
-                'joinType' => self::INNER_JOIN,
+                'joinType' => self::LEFT_JOIN,
```

Comment:
INNER_JOIN was a mistake, we are checking for a value in WHERE. We can't inner join if the table is in an OR clause.